client/js/components/GameView/MoveChoiceOverlay.vue

```diff
@@ -70,181 +70,167 @@ export default {
-        scuttleDisabledExplanation = allMovesDisabledExplanation;
+      if (this.allMovesAreDisabled) {
+        scuttleDisabledExplanation = this.disabledText;
       }
-      const scuttleMove = {
+      return {
         displayName: 'Scuttle',
         eventName: 'scuttle',
         moveDescription: 'Scrap a lower point card',
-        disabled: scuttleDisabled || allMovesAreDisabled,
+        disabled: scuttleDisabled || this.allMovesAreDisabled,
         disabledExplanation: scuttleDisabledExplanation,
       };
+    },
+    oneOffMove() {
+      return {
+        displayName: 'One-Off',
+        eventName: 'oneOff',
+        moveDescription: this.selectedCard.ruleText,
+        disabled: this.allMovesAreDisabled,
+        disabledExplanation: this.disabledText,
+      };
+    },
+
+    targetedOneOffMove() {
+      let oneOffDisabled = !this.isPlayersTurn;
+      let oneOffDisabledExplanation = "It's not your turn";
+      if (this.isPlayersTurn) {
+        if (this.opponentQueenCount >= 2) {
+          oneOffDisabled = true;
+          oneOffDisabledExplanation = `You can't play a ${this.selectedCard.rank} while your opponent has two or more queens`;
+        } else {
+          let validTargetExists;
+          // Twos
+          if (this.selectedCard.rank === 2) {
+            const numOpFaceCards = this.$store.getters.opponent.faceCards.length;
+            const numOpJacks = this.$store.getters.opponent.points.reduce(
+              (jackCount, pointCard) => {
+                return jackCount + pointCard.attachments.length;
+              },
+              0
+            );
+            const numTotalTargets = numOpFaceCards + numOpJacks;
+            validTargetExists = numTotalTargets >= 1;
+            if (this.allMovesAreDisabled) {
+              oneOffDisabledExplanation = this.disabledText;
+            } else if (!validTargetExists) {
+              oneOffDisabled = true;
+              oneOffDisabledExplanation = 'There are no Royals to target';
+            }
+            // Nines
+          } else {
+            const numValidTargets =
+              this.$store.getters.opponent.points.length +
+              this.$store.getters.opponent.faceCards.length;
+            if (this.allMovesAreDisabled) {
+              oneOffDisabledExplanation = this.disabledText;
+            } else if (numValidTargets === 0) {
+              oneOffDisabled = true;
+              oneOffDisabledExplanation = 'There are no point cards or Royals to target';
+            }
+          }
+        }
+      }
+      return {
+        displayName: 'One-Off',
+        eventName: 'targetedOneOff',
+        moveDescription: this.selectedCard.ruleText,
+        disabled: oneOffDisabled || this.allMovesAreDisabled,
+        disabledExplanation: oneOffDisabledExplanation,
+      };
+    },
+    jackMove() {
+      let ableToJack = false;
+      let disabledExplanation = '';
+      if (this.isPlayersTurn && !this.allMovesAreDisabled) {
```

Comment:
`allMovesAreDisabled` already checks if it's your turn so we can just
```suggestion
      if (!this.allMovesAreDisabled) {
```